diagnostics/collect-networking-logs.ps1

```diff
@@ -30,7 +30,9 @@ if (Test-Path $wslconfig)
 if ($RestartWslReproMode)
 {
     # The WSL HNS network is created once per boot. Resetting it to collect network creation logs.
+    # Note: The below HNS commands apply only to WSL in NAT mode
     Get-HnsNetwork | Where-Object {$_.Name -eq 'WSL'} | Remove-HnsNetwork
+    Get-HnsNetwork | Where-Object {$_.Name -eq 'WSL (Hyper-V firewall)'} | Remove-HnsNetwork

```

Comment:
nit: could be merged with the above call by: 
`    Get-HnsNetwork | Where-Object {$_.Name -eq 'WSL' -Or $_.Name -eq 'WSL (Hyper-V firewall)' } | Remove-HnsNetwork

Reply:
thanks, fixed